IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers/WolfSSLDemo/src/wolfDemo/wolf_console.c

```diff
@@ -0,0 +1,142 @@
+/* wolf_main.c
+ *
+ * Copyright (C) 2006-2020 wolfSSL Inc.
```

Comment:
Fix date

Reply:
Fixed and committed.(I reviewed all files.)